app/models/game.rb

```diff
@@ -6,7 +6,11 @@ class Game < ActiveRecord::Base
   belongs_to :black_player, class_name: 'User'
   belongs_to :white_player, class_name: 'User'
 
-  after_create :populate_board!
+  def self.create_and_populate_board!(params)
+    create(params)
+    populate_board!
+    self
+  end
```

Comment:
My fault, I wrote this out wrong on our call :)

Should be...
```Ruby
 def self.create_and_populate_board!(params)
    new_game = create(params)
    new_game.populate_board!
    new_game
  end
```
Since that method is a `class` method (`self.`). We have to call populate_board! on the new instance of `Game` that we created.

Rest looks great, go ahead and merge whenever ready :)